smarts/core/remote_agent_buffer.py

```diff
@@ -19,82 +19,62 @@
 # THE SOFTWARE.
 import logging
 import atexit
+from concurrent import futures
 
 from threading import Lock, Thread
 
 from .remote_agent import RemoteAgent, RemoteAgentException
 
 
 class RemoteAgentBuffer:
-    def __init__(self, buffer_size=1):
+    def __init__(self, buffer_size=3):
```

Comment:
I think this deserves a doc string mentioning that `buffer_size` should be greater than 0.